components/Layout.tsx

```diff
@@ -54,7 +54,7 @@ function Header(props: JSX.HTMLAttributes<HTMLElement>) {
       {...props}
       class={`p-4 justify-between ${SITE_WIDTH_STYLES} flex z-10 ${
         props.class ?? ""
-      }`}
+      } dark:text-white`}
```

Comment:
Are you able to move this next to the other hardcoded classes? It would look a little cleaner.